scripts/dockerbot/dockerbot.py

```diff
@@ -420,14 +402,11 @@ def enc_key(self):
     def create_pytest_envvars(self):
         self.env_details = {'BROWSER': self.args['browser'],
                             'CFME_CRED_REPO': self.args['cfme_cred_repo'],
-                            'CFME_CRED_REPO_DIR': self.args['cfme_cred_repo_dir'],
```

Comment:
The original intention of having the two was so that the base could be different from the PR branch, thereby allowing you to test against your own repo, I'm guessing this is no longer possible with this change?